packages/openapi-v3/src/__tests__/unit/decorators/tags.decorator.unit.ts

```diff
@@ -0,0 +1,141 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {anOpenApiSpec} from '@loopback/openapi-spec-builder';
+import {expect} from '@loopback/testlab';
+import {getControllerSpec} from '../../..';
+import {oas} from '../../../decorators';
```

Comment:
This should be merged into the `import` as follows:
```ts
import {getControllerSpec, oas} from '../../..';
```

Reply:
:+1:, guess auto-import didn't do it's thing correctly.